docs/cli.md

```diff
@@ -0,0 +1,447 @@
+                  
+
+# BigFlow CLI
+
+BigFlow offers a command-line tool called `bf`.
+It lets you run, build, and deploy your workflows from command-line on any machine with Python.
+
+`bf` is a recommended way of working with BigFlow
+for developing on a local machine as well as for build and deployment automation on CI/CD servers.  
+
+## Installing `bf`
+
+Prerequisites:
+
+1. [Python](https://www.python.org/downloads/) 3.7
+2. [Google Cloud SDK](https://cloud.google.com/sdk/docs/downloads-interactive)  
+
+
+You can install the `bf` tool globally but we recommend to 
+install it locally with `venv`, in your project's folder:
+
+```bash
+python -m venv .bigflow_env
+source .bigflow_env/bin/activate
+cd .bigflow_env
+```
+
+Install the `bf` tool:
+
+```bash
+pip install bigflow
+```
+
+Test it:
+
+```shell
+bf -h
+```
+
+You should see the welcome message and the list of all `bf` commands:
+
+```text
+Welcome to BiggerQuery CLI. Type: bf {run,deploy-dags,deploy-
+image,deploy,build,build-dags,build-image,build-package} -h to print detailed
+help for a selected command.
+```
+
+Each command has its own set of arguments. Check it with `-h`, for example:
+
+```shell
+bf run -h
+```
+
+## Running jobs and workflows
+
+
+`bf run` command lets you run a job or a workflow for a given `runtime`,
+
+It simply takes your local source code and runs it directly on GCP, without deploying to
```

Comment:
runs it on your local machine

"runs it directly on GCP" sounds like it takes the local source code and runs it on GCP